spec/v2/index.bs

```diff
@@ -621,21 +621,23 @@ Advisement:
       <tr>
         <td><dfn>primaryDataShare</dfn> : [=Percent=]
         <td>Number
-        <td>O*
+        <td>O
```

Comment:
Why should this be O rather than O*? Per the framework (page 64), primary data share is mandatory "from 2025 onwards". Therefore to align with the framework, I think this attribute should be O*. If this isn't just a typo though, let me know, and we can raise this in our weekly sync with Joy and Prasad on Wednesday.

Reply:
Addressed this via `a464d9503a2e5cb83e9661cdb14940ad94568900`